src/displayapp/DisplayAppRecovery.cpp

```diff
@@ -24,7 +25,8 @@ DisplayApp::DisplayApp(Drivers::St7789& lcd,
                        Pinetime::Controllers::TimerController& timerController,
                        Pinetime::Controllers::AlarmController& alarmController,
                        Pinetime::Controllers::BrightnessController& brightnessController,
-                       Pinetime::Controllers::TouchHandler& touchHandler)
+                       Pinetime::Controllers::TouchHandler& touchHandler,
+                       Pinetime::Controllers::FS& filesystem)
```

Comment:
we're using `fs` for the filesystem controller everywhere else (I think). I like the explicitness of `filesystem`

would you suggest to use `filesystem` instead of `fs` in the future?

Reply:
Yes, I agree with you, `filesystem` is more explicit!